client/app/dedicatedCloud/components/activationStatus/constants.js

```diff
@@ -0,0 +1,33 @@
+export const COMPONENT_NAME = 'activationStatus';
+export const CONSTANT_NAME = 'ACTIVATION_STATUS';
+export const MODULE_NAME = 'activationStatus';
```

Comment:
same here, i think it's unecessary to export a module name as a constant. constants should contains "application" logic constants, not module name declaration